cmd/buffer-api/main.go

```diff
@@ -150,7 +150,7 @@ func run() error {
 			server := bufferGenSvr.New(endpoints, c.Muxer, c.Decoder, c.Encoder, c.ErrorHandler, c.ErrorFormatter, docsFs, docsFs, docsFs, docsFs, swaggerUiFs)
 
 			// Mount endpoints
-			server.Use(middleware.TraceEndpoints())
+			server.Use(middleware.OpenTelemetryExtractEndpoint())
```

Comment:
Telementria HTTP servera pozostava z 3 middlewares:
- middleware.OpenTelemetry (kazdy request)
- middleware.OpenTelemetryExtractRoute (ak sa najde route)
- middleware.OpenTelemetryExtractEndpoint (goa endpoint)
  - ^^^ sa povodne volalo `TraceEndpoints`, premenoval som aj subor `endpoint.go` -> `otel_endpoint.go`.
  
  Je to vsetko sucast jednej telemetrie, tak by to malo davat vacsi zmysel.